tests/end-to-end/backup/pkg/tests/eventmesh/event.go

```diff
@@ -17,7 +21,51 @@ import (
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 )
 
-func SendEvent(target, payload, eventType, eventTypeVersion string) error {
+type LegacyEvent struct {
+	EventType        string    `json:"event-type"`
+	EventTypeVersion string    `json:"event-type-version"`
+	EventID          string    `json:"event-id"`
+	EventTime        time.Time `json:"event-time"`
+	Data             string    `json:"data"`
+}
+
+func SendLegacyEvent(target, payload, eventType, eventTypeVersion string) error {
```

Comment:
(nit) mhh ... for people who are not aware of that legacy is, a name like SendEventCompatibilityLayer would help I think. It clearly states where the event is sent to. Same for `LegacyEvent`. What about `EventBusV1` ? Not sure which version you are using.

```suggestion
func SendEventToCompatibilityLayer(target, payload, eventType, eventTypeVersion string) error {
```

Reply:
the target can be specified ... compatibilitylayer is just one target that works with this function